chef/cookbooks/network/recipes/default.rb

```diff
@@ -509,4 +511,26 @@ def kill_nic(nic)
       end
     end
   end
+
+  if ovs_setup_once
+    # If we're using an ovs-bridge somewhere, enable wicked-nanny to be started
+    # for the next boot.
+    # Note: There's no need to restart wicked here as all interfaces
+    # should be correctly configure at this point.
+    template "/etc/wicked/local.conf" do
+      source "wicked-local.conf.erb"
+      owner "root"
+      group "root"
+      mode "0644"
+      variables(
+        use_nanny: true
+      )
```

Comment:
@vuntz Hm, we are already in the  ```when "suse"``` branch of ```case node["platform"]``` here. So I guess only the ```if !node[:platform_version] < 12.0``` would make sense here, right? Or probably better: ```node[:platform_version] >= 12.0```